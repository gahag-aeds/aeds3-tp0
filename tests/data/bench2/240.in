240 120
63 76 35 21 95 8 18 88 27 60 61 38 73 78 72 17 99 36 58 62 0 82 24 2 79 43 12 55 57 83 20 73 59 7 94 6 15 12 95 94 25 8 85 50 38 57 67 37 93 77 51 46 11 75 48 90 70 12 45 28 47 17 1 59 76 47 65 91 60 12 38 37 20 23 87 10 80 7 99 26 36 2 72 48 78 72 38 0 36 36 80 84 5 33 95 82 81 60 25 93 73 63 30 45 86 17 8 19 76 7 45 13 10 69 61 40 41 51 40 77 87 73 13 45 6 8 27 87 21 52 80 46 68 62 43 54 32 51 25 8 59 70 73 21 39 34 61 32 38 53 10 77 26 75 22 85 36 1 72 57 6 5 55 74 67 98 80 99 50 6 60 61 76 33 82 68 20 95 0 58 0 62 35 79 90 10 64 26 63 88 35 69 93 90 95 61 88 76 12 38 82 72 51 10 6 85 30 78 32 83 36 33 45 23 64 35 85 28 61 49 16 96 18 62 86 14 75 27 42 87 17 76 60 69 38 18 6 69 96 39 
Add 113 192
Min 45 124
Add 65 144
Max 70 149
Sub 80 159
Min 89 168
Sub 58 137
Sum 146 225
Add 149 228
Sum 19 98
Add 115 194
Sum 117 196
Sub 28 107
Min 65 144
Add 159 238
Max 26 105
Add 11 90
Sum 14 93
Add 65 144
Min 5 84
Sub 122 201
Max 123 202
Add 61 140
Sum 54 133
Sub 104 183
Sum 60 139
Sub 144 223
Sum 10 89
Add 121 200
Max 52 131
Add 21 100
Max 105 184
Sub 96 175
Sum 85 164
Sub 14 93
Min 15 94
Add 45 124
Max 71 150
Sub 160 239
Min 65 144
Sub 122 201
Sum 146 225
Sub 22 101
Min 21 100
Add 83 162
Max 20 99
Sub 78 157
Min 93 172
Sub 34 113
Min 143 222
Sub 130 209
Min 27 106
Add 105 184
Sum 30 109
Sub 80 159
Sum 80 159
Add 89 168
Min 95 174
Add 89 168
Min 81 160
Sub 146 225
Sum 110 189
Add 25 104
Sum 81 160
Sub 66 145
Max 39 118
Add 127 206
Min 111 190
Add 141 220
Max 159 238
Add 15 94
Sum 102 181
Sub 144 223
Max 68 147
Sub 154 233
Max 66 145
Sub 88 167
Min 72 151
Add 31 110
Sum 153 232
Sub 54 133
Sum 160 239
Add 19 98
Sum 30 109
Add 61 140
Sum 130 209
Add 109 188
Sum 22 101
Sub 64 143
Min 69 148
Sub 102 181
Min 49 128
Sub 50 129
Max 127 206
Add 1 80
Min 116 195
Add 37 116
Min 127 206
Sub 98 177
Max 49 128
Add 125 204
Sum 144 223
Add 151 230
Min 141 220
Sub 52 131
Sum 16 95
Sub 78 157
Max 139 218
Add 87 166
Min 140 219
Add 3 82
Min 12 91
Add 139 218
Min 21 100
Add 73 152
Min 72 151
Sub 150 229
Sum 53 132
Add 93 172
Max 53 132
