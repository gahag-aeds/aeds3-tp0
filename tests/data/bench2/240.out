0
100
1
4245
3859
4257
-2
98
3826
0
102
3902
3809
3917
101
101
3759
1
102
2
4169
1
101
0
0
0
3890
3632
0
-1
3824
3818
99
0
101
3802
100
100
-1
4111
4165
4068
4070
4150
0
0
99
-1
-1
102
4009
0
4018
100
1
-1
-1
0
3801
102
